lean4-infoview/src/infoview/interactiveCode.tsx

```diff
@@ -145,10 +151,39 @@ const HoverableTypePopupSpan =
       tippyInstance.current.show()
   }
 
+  const insideTip = (e : Event) => {
+    // Don't hide the tips if the mouse click is inside another tip!
+    var span : any = e.target;
+    if (span._tippy) {
+      return true;
+    }
+    return false;
+  }
+
   return (
     <LazyTippy
       ref={ref}
+      // To avoid weird layout bugs where Chrome does not like a <div> popping up
```

Comment:
Are we sure that this is a Chrome bug rather than some weird edge case in the HTML layout specification, in which case it would also misbehave in other browser engines? If we're not, I'd just write we're working around a layout issue.

Reply:
I removed the "bug" language